apps/docs/components/Favicons.tsx

```diff
@@ -22,15 +22,15 @@ const Favicons = () => {
       {/* prettier-ignore */}
       <link rel="apple-touch-icon-precomposed" sizes="152x152" href={`${basePath}/favicon/apple-touch-icon-152x152.png`}/>
       {/* prettier-ignore */}
-      <link rel="icon" type="image/png" href={`${basePath}/favicon/favicon-196x196.png" sizes="196x196`}/>
```

Comment:
sure about this @fsansalvadore ?

Reply:
hmmm it was showing an error in console about those but I can't reproduce it anymore... so I'll revert this for now

Reply:
Sorry, there actually was something wrong. Just fixed the quotes now.